ui/src/app/components/help-sidebar.tsx

```diff
@@ -397,37 +395,23 @@ export const HelpSidebar = fp.flow(
     }
 
     setActiveIcon(activeIcon: string) {
-      this.setState({activeIcon});
-      if (activeIcon) {
-        localStorage.setItem(LOCAL_STORAGE_KEY_SIDEBAR_STATE, activeIcon);
-      } else {
-        localStorage.removeItem(LOCAL_STORAGE_KEY_SIDEBAR_STATE);
-      }
+      setSidebarActiveIconStore.next(activeIcon);
     }
 
     async componentDidMount() {
-      const lastSavedKey = localStorage.getItem(LOCAL_STORAGE_KEY_SIDEBAR_STATE);
-
-      // This is a little hacky but it's necessary because
-      // 1. the pageKey is needed to know which icons are available on this page but it's not always available on mount
-      // 2. router events (which usually close the sidebar panel) during initial page load will update the activeIcon
-      //    and overwrite the value stored in localStorage key so we need to "save" it here
-      // I'd like to clean this up but I think it'll have to wait until the router migration is complete.
-      this.loadLastSavedKey = (() => {
-        let loadedLastSavedKey = false;
-        return () => {
-          if (!loadedLastSavedKey && this.props.pageKey) {
-            const iconConfig = this.icons().find(icon => icon.id === lastSavedKey);
-            setSidebarActiveIconStore.next(iconConfig ? iconConfig.id : null);
-            loadedLastSavedKey = true;
-          }
-        };
-      })();
-
-      this.loadLastSavedKey();
-      this.subscription = participantStore.subscribe(participant => this.setState({participant}));
-      this.subscription.add(setSidebarActiveIconStore.subscribe(activeIcon => {this.setActiveIcon(activeIcon); }));
-      this.subscription.add(routeDataStore.subscribe((newRoute, oldRoute) => {
+      // This is being set here instead of the constructor since I like showing the opening animation of the side
```

Comment:
super nit: would generally avoid first-person in comments, e.g. "since I like showing" -> "in order to show", unless it's a temporary TODO comment.